### PR TITLE
Firebase/messaging

### PR DESCRIPTION
**Pull request by devcsomnicg**
## JIRA Ticket(s) if any

* [MOB-XXXX](https://iterable.atlassian.net/browse/MOB-XXXX)

## Test Steps

### CODE REVIEW

**Review by mprew97 — COMMENTED** [review]
.

File: src/message/utils.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      }
+    });
+  };
+  
+  private async registerBrowserToken (token: string) {
+    return baseIterableRequest<IterableResponse>({
+      method: 'POST',
+      url: REGISTER_TOKEN,
+      headers: {
+        "Api-Key": "1d925bbf54f34bcb92a15da14bfb5d1d"
```
Comment (mprew97) [inline src/message/utils.ts:63]: Need to remove the hardcoded api key here from the commit history and make this use the environment variable in `.env.example`.

Comment (devcsomnicg) [reply src/message/utils.ts:63]: Ok

Comment (devcsomnicg) [reply src/message/utils.ts]: Actually, we added the "initialize(process.env.API_KEY)" call on the example app, to add "API_KEY" for API calls, that's why we removed the static header

**Pull request closed without merging**